docker/deploy.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -euo pipefail
+
+sudo apt-get update && sudo apt-get install -y python-pip && sudo pip install awscli
+
+# Get login token and execute login
+$(aws ecr get-login --no-include-email --region $AWS_REGION)
+echo -e $GITLAB_PRIVATE_KEY > .ssh/id_rsa
```

Comment:
Sry, my admin knowledge is soo bad:
What prevents me from downloading the docker image, run it and log in and simply read the .ssh/id_rsa file?

Reply:
Ah, we never make the docker image public! I see.

Reply:
Right, we should probably delete the ssh file from the built image though just in case it ever leaks.